specification/security/resource-manager/Microsoft.Security/preview/2023-05-01-preview/healthReports.json

```diff
@@ -0,0 +1,373 @@
+        ],
+        "description": "Get health report of resource",
+        "operationId": "HealthReports_Get",
+        "parameters": [
+          {
+            "$ref": "../../../../../common-types/resource-management/v5/types.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "../../../common/v1/types.json#/parameters/ResourceId"
+          },
+          {
+            "$ref": "#/parameters/HealthReportName"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/HealthReport"
+            }
+          },
+          "default": {
+            "description": "Common error response for all Azure Resource Manager APIs to return error details for failed operations.",
+            "schema": {
+              "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "HealthReportsList": {
+      "type": "object",
+      "description": "Page of health reports list",
+      "properties": {
+        "value": {
+          "description": "Collection of health reports in this page",
+          "readOnly": true,
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/HealthReport"
+          }
+        },
+        "nextLink": {
+          "readOnly": true,
+          "type": "string",
+          "description": "The URI to fetch the next page"
+        }
+      }
+    },
+    "HealthReport": {
+      "type": "object",
+      "description": "The health report resource",
+      "properties": {
+        "properties": {
+          "description": "Properties of a health report",
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/HealthReportProperties"
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "../../../common/v1/types.json#/definitions/Resource"
+        }
+      ]
+    },
+    "HealthReportProperties": {
+      "type": "object",
+      "description": "Describes properties of the health report",
+      "properties": {
+        "resourceDetails": {
+          "$ref": "#/definitions/resourceDetails"
+        },
+        "environmentDetails": {
+          "$ref": "#/definitions/environmentDetails"
+        },
+        "healthDataClassification": {
+          "$ref": "#/definitions/healthDataClassification"
+        },
+        "status": {
+          "$ref": "#/definitions/status"
+        },
+        "affectedDefendersPlans": {
+          "type": "array",
+          "description": "The affected defenders plans by unhealthy report",
+          "items": {
+            "type": "string",
+            "description": "plan name"
+          }
+        },
+        "affectedDefendersSubPlans": {
+          "type": "array",
+          "description": "The affected defenders sub plans by unhealthy report",
```

Comment:
>sub plans by [](http://example.com/codeflow?start=49&length=12)

RE 'sub plans' - you should expand out acronyms and abbreviations in docs, writing in a universally accessible  (requiring little specialised knowledge) conversational style.

Reply:
While I understand your concern - 'sub' is not an acronym or abbreviation in this specific context.  
It refers to a subsection part of a larger plan.

Reply:
So, its an abbreviation of 'subsection' then, no?